addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -98,6 +99,7 @@ def _make_request(self, url, params=False):
             error_code = proxy_error['code']
             if error_code == 'refresh_token_expired':
                 self._renew_token()
+                self.env.cr.commit() # We do not want to lose it if in the _make_request below something goes wrong
```

Comment:
I didn't check anything yet, but you need to make sure that nothing is committed during tests, same in `addons/l10n_it_edi_sdicoop/models/account_edi_format.py`